docs/compute.md

```diff
@@ -309,6 +309,24 @@ With this simple but first initial algorithm, we managed to mitigate automatical
 
 You have more than 2 hosts? Again, same idea, it will also work.
 
+## VM anti-affinity
+
+VM anti-affinity is a feature that prevents VMs with the same user tags to run on the same host. This functionality is available directly in the load-balancer plugin.
+This way, you will avoid having pairs of redundant VMs or similar put on the same host.
+
+Let's see a simple example: you have multiple VMs using MySQL and Postgresql with high availability/replication. Obviously, you don't want to lose the replicated database inside VMs on the same physical host. Just create your plan like this:
+
+![](https://xen-orchestra.com/blog/content/images/2021/03/antiaffinity.png)
```

Comment:
It would probably be more future proof to copy the image locally in this repository.

Reply:
There are already a lot of links existing in the doc linked to the blog, I don’t know. In this case, it would be interesting to change all links. ¯\_(ツ)_/¯